---
title: "beta diversity weighted UniFrac and Bray Curtis"
author: "Rosalane"
date: '2022-09-02'
output: html_document
---
Library needed
```{r}
library("phyloseq")
library("ggplot2")
library("ape")
library("gridExtra")
library("ggpubr")
library("ggsignif")
library(forcats)
library(vegan)
library(cowplot)
library(tidyverse)
theme_set(theme_cowplot())
set.seed(7)
source("C:/users/kithanro/Documents/16s_Gala_postharvest_2021/DenefLab-MicrobeMiseq/R/miseqR.R")
```


```{r setup, include=FALSE}
knitr::opts_chunk$set(echo = TRUE)
knitr::opts_knit$set(root.dir = "C:/Users/kithanro/Documents/16s 2020 data")
```

#Phyloseq object
```{r echo=TRUE}
#read in otu table
otu_table <- read.table("OTU_table.txt",
                        sep="\t",row.names = 1,header = TRUE) %>%
  as.matrix(otu_table,rownames=TRUE)

#read in taxonomy
taxonomy <- read_delim("taxonomy.csv", delim = ",", col_names = TRUE) %>% 
  column_to_rownames(var = "OTU_ID") %>%
  as.matrix()

#read in metadata
metadata=read.table("Sample_metadata.2020.csv", sep = ",", header = TRUE,
                    row.names=1, na.strings = "") 
metadata
phy_tree = read_tree("tree.nwk")

#import as phyloseq objects
OTU = otu_table(otu_table,taxa_are_rows=TRUE)
TAX = tax_table(taxonomy)
META = sample_data(metadata)
#Physeq object
physeq <- phyloseq(OTU,TAX,META,phy_tree) #454 taxa and 361 samples 
```

#Fitering Chloroplast and Mitocondria OTUs
```{r}
#physeq <- physeq_filtered

test_df <- as.data.frame(sample_sums(physeq)) %>% #sum reads per sample
  rownames_to_column(var = "Sample_ID")

test_plot <- ggplot(test_df, aes(x = reorder(Sample_ID, -`sample_sums(physeq)`), #orders axis
                                 y = `sample_sums(physeq)`
                                 )) + 
  geom_col() + ylab("Reads per sample") + xlab("Sample_ID") + #clean up labels
  theme(axis.text.x = element_text(angle = -90, hjust=0))
test_plot

physeq_filtered <- physeq %>% 
  
  subset_taxa(Order != " Chloroplast") %>%
  subset_taxa(Family != " Mitochondria") #313 taxa and 361 samples

#check reads per sample
test_df <- as.data.frame(sample_sums(physeq_filtered)) %>% #sum reads per sample
  rownames_to_column(var = "Sample_ID") #format for plot

# make the plot
test_plot <- ggplot(test_df, aes(x = reorder(Sample_ID, -`sample_sums(physeq_filtered)`), #orders axis
                                 y = `sample_sums(physeq_filtered)`
                                 )) + 
  geom_col() + ylab("Reads per sample") + xlab("Sample_ID") + #clean up labels
  theme(axis.text.x = element_text(angle = -90, hjust=0))
test_plot

# more filtering to remove samples that didn't sequence well or OTUs that may not be real 
  physeq_filtered2 <- physeq_filtered %>%  
  prune_samples(sample_sums(.) >= 100, .) %>% #remove samples with <=100 reads per samples    #313 taxa and 279 samples
  prune_taxa(taxa_sums(.) >= 2, .) # remove taxa with <=2 reads per OTU #300 taxa and 279 samples
  
  # for prunning 5 reads taxa #153 taxa and 279 samples
  
# subset samples to each read numbers 
  
#extrct sample df from phylose object
test_samp <- sample_data(physeq_filtered2)

# #extrct sample df from phylose object
# test_samp <- sample_data(physeq_filtered2)
# 
# #To store phyloseq object in our local computer
# saveRDS(physeq_filtered2, file = "C:/Users/kithanro/Documents/16s_Gala_postharvest_2021/physeq_filtered2.RDS") 
# 
# #To restore back the phyloseq object from local computer to R
# 
# #object name. eg. physeq_filtered2 <- readRDS(file = "C:/Users/kithanro/Documents/16s_Gala_postharvest_2021/physeq_filtered2.RDS")
```


##Beta Diversity by tissuetypes pulpvs. skin in Gala (Filter Rotten and day 42)

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Jonathan") %>%
  #subset_samples(Tissue_types != "Skin")  %>%
  subset_samples(Timepoints != "Rotten")  %>%
  subset_samples(Timepoints != "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  #subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Cultivar == "Jonathan")  
  

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue.types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue.types", color="Tissue.types", shape="Tissue.types", title="PCoA Weighted Unifrac- Gala") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue.types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.14, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue.types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (47.1%) ", y = "PCoA 2 (19.5%)", colour = "Tissue.types", fill = "Tissue.types", title="PCoA Weighted Unifrac- Gala") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```
##Beta Diversity by tissuetypes pulp vs. skin in Gala (day 42)

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  subset_samples(Cultivar != "Jonathan") %>%
  #subset_samples(Tissue_types != "Skin")  %>%
  subset_samples(Timepoints != "Rotten")  %>%
  subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Cultivar == "Gala")  
  

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue.types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Tissue.types, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue.types", color="Tissue.types", shape="Tissue.types", title="PCoA Weighted Unifrac- Gala day 14 and 28") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue.types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.14, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue.types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (31.3%) ", y = "PCoA 2 (25.9%)", colour = "Tissue.types", fill = "Tissue.types", title="PCoA Weighted Unifrac- Gala day 42") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```



#Beta Diversity by Timepoints in Gala Pulp

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  subset_samples(Cultivar != "Jonathan") %>%
  subset_samples(Tissue.types != "Skin")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Cultivar == "Gala")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac- Gala Pulp") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.14, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (70.5%) ", y = "PCoA 2 (19.2%)", colour = "Timepoints", fill = "Timepoints", title="PCoA Weighted Unifrac- Gala Pulp") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in Gala Skin

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  subset_samples(Cultivar != "Jonathan") %>%
  subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Cultivar == "Gala")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac- Gala Skin") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.37, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (33.3%) ", y = "PCoA 2 (18.7%)", colour = "Timepoints", fill = "Timepoints", title="PCoA Weighted Unifrac- Gala Skin") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```
##Beta Diversity by tissuetypes pulp vs. skin in Jonathan

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  subset_samples(Cultivar != "Gala") %>%
  #subset_samples(Tissue_types != "Skin")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints != "Day 28")  %>%
  # subset_samples(Timepoints != "Day 42")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Cultivar == "Jonathan")  
  

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue.types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Tissue.types, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue.types", color="Tissue.types", shape="Tissue.types", title="PCoA Weighted Unifrac- Jonathan") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue.types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.14, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue.types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (41.3%) ", y = "PCoA 2 (23.3%)", colour = "Tissue.types", fill = "Tissue.types", title="PCoA Weighted Unifrac- Jonathan") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in Jonathan Skin

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  #subset_samples(Tissue.types != "Pulp")  %>%
  subset_samples(Timepoints != "Day 28")  %>%
  subset_samples(Timepoints != "Day 14")  %>%
  subset_samples(Timepoints != "Day 0")  %>%
  #subset_samples(Timepoints == "Day 14")  %>%
  subset_samples(Tissue.types == "Skin")  %>%
  #subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Cultivar == "Jonathan")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac- Gala Skin") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.18, ", paste(italic('p')),"=0.03"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (33.3%) ", y = "PCoA 2 (18.1%)", colour = "Timepoints", fill = "Timepoints", title="PCoA Weighted Unifrac- Jonathan Skin. Day 42 vs Rotten") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in Jonathan Skin day 0 and day 14

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints == "Day 14")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 0" | Timepoints == "Day 14") %>%
  subset_samples(Cultivar == "Jonathan")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac- Gala Skin day 0 and day 14") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.14, ", paste(italic('p')),"=0.002"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (38%) ", y = "PCoA 2 (29.9%)", colour = "Timepoints", fill = "Timepoints", title="PCoA Weighted Unifrac- Jonathan Skin: Day 0 and day 14") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in Jonathan Skin day 14 and day 28

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints == "Day 14")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 14" | Timepoints == "Day 28") %>%
  subset_samples(Cultivar == "Jonathan")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac- Gala Skin day 14 and day 28") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.18, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (31.8%) ", y = "PCoA 2 (24.2%)", colour = "Timepoints", fill = "Timepoints", title="PCoA Weighted Unifrac- Jonathan Skin: Day 14 and day 28") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in Jonathan Skin day 28 and day 42

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints == "Day 14")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 28" | Timepoints == "Day 42") %>%
  subset_samples(Cultivar == "Jonathan")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac- Gala Skin day 28 and day 42") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.14, ", paste(italic('p')),"=0.003"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (31.8%) ", y = "PCoA 2 (24.2%)", colour = "Timepoints", fill = "Timepoints", title="PCoA Weighted Unifrac- Jonathan Skin: Day 28 and day 42") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in Jonathan Skin day 42 and Rotten

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints == "Day 14")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 42" | Timepoints == "Rotten") %>%
  subset_samples(Cultivar == "Jonathan")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac- Gala Skin day 42 and Rotten") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.18, ", paste(italic('p')),"=0.038"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (50.5%) ", y = "PCoA 2 (22.9%)", colour = "Timepoints", fill = "Timepoints", title="PCoA Weighted Unifrac- Jonathan Skin: Day 42 and day Rotten") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```
#Beta Diversity by Tissue types in Jonathan Rotten

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  #subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints == "Day 14")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Rotten") %>%
  subset_samples(Cultivar == "Gala")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue.types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Tissue.types, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue.types", color="Tissue.types", shape="Timepoints", title="PCoA Weighted Unifrac- Jonathan Rotten") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue.types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.13, ", paste(italic('p')),"=0.2"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue.types", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (73.4%) ", y = "PCoA 2 (14.5%)", colour = "Tissue.types", fill = "Tissue.types", title="PCoA Weighted Unifrac- Gala. Skin vs. Pulp. Rotten") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Tissue types in Jonathan day 42

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  subset_samples(Cultivar != "Gala") %>%
  #subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints == "Day 14")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 42") %>%
  subset_samples(Cultivar == "Jonathan")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue.types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue.types", color="Tissue.types", shape="Timepoints", title="PCoA Weighted Unifrac- Jonathan Day 42") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue.types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.13, ", paste(italic('p')),"=0.02"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue.types", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (53.9%) ", y = "PCoA 2 (24.9%)", colour = "Tissue.types", fill = "Tissue.types", title="PCoA Weighted Unifrac- Jonathan Day 42") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```
#Beta Diversity by Tissue types in Gala day 42

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  #subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints == "Day 14")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 42") %>%
  subset_samples(Cultivar == "Gala")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue.types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue.types", color="Tissue.types", shape="Timepoints", title="PCoA Weighted Unifrac- Gala Day 42") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue.types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.34, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue.types", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (57.6%) ", y = "PCoA 2 (27.5%)", colour = "Tissue.types", fill = "Tissue.types", title="PCoA Weighted Unifrac- Gala Day 42") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Tissue types in Gala Rotten

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  #subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints == "Day 14")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Rotten") %>%
  subset_samples(Cultivar == "Gala")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue.types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue.types", color="Tissue.types", shape="Timepoints", title="PCoA Weighted Unifrac- Gala Rotten") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue.types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.24, ", paste(italic('p')),"=0.4"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue.types", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (62.6%) ", y = "PCoA 2 (32.3%)", colour = "Tissue.types", fill = "Tissue.types", title="PCoA Weighted Unifrac- Gala Rotten") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```




#Beta Diversity by Timepoints in Jonathan Pulp

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Skin")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints == "Day 42")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Cultivar == "Jonathan")  


test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac- Gala Pulp") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.76, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas", ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (54.6%) ", y = "PCoA 2 (28.2%)", colour = "Timepoints", fill = "Timepoints", title="PCoA Weighted Unifrac- Jonathan Pulp") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```


##Beta Diversity Jonathan vs. Gala skin Day 0

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints != "Day 42")  %>%
  # subset_samples(Timepoints != "Day 28")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 14") %>%
  subset_samples(Cultivar == "Jonathan" | Cultivar == "Gala")  

test_samp <- sample_data(physq_sub)

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Cultivar)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Cultivar, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Cultivar", color="Cultivar", shape="Tissue_types", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Skin Day 14") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Cultivar") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.26, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Cultivar", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Skin Day 14", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (44.8%) ", y = "PCoA 2 (22.2%)", colour = "Cultivar", fill = "Cultivar") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```
##Beta Diversity Jonathan vs. Gala skin Day 28

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints != "Day 42")  %>%
  # subset_samples(Timepoints != "Day 28")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 28") %>%
  subset_samples(Cultivar == "Jonathan" | Cultivar == "Gala")  

test_samp <- sample_data(physq_sub)

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Cultivar)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Cultivar, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Cultivar", color="Cultivar", shape="Tissue_types", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Skin Day 28") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Cultivar") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.42, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Cultivar", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Skin Day 28", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (51.9%) ", y = "PCoA 2 (17.5%)", colour = "Cultivar", fill = "Cultivar") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

##Beta Diversity Jonathan vs. Gala skin Day 42

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints != "Day 42")  %>%
  # subset_samples(Timepoints != "Day 28")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 42") %>%
  subset_samples(Cultivar == "Jonathan" | Cultivar == "Gala")  

test_samp <- sample_data(physq_sub)

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Cultivar)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Cultivar, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Cultivar", color="Cultivar", shape="Tissue_types", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Skin Day 42") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Cultivar") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.17, ", paste(italic('p')),"=0.005"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Cultivar", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Skin Day 42", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (33.6%) ", y = "PCoA 2 (30%)", colour = "Cultivar", fill = "Cultivar") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

##Beta Diversity Jonathan vs. Gala skin Day Rotten

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints != "Day 42")  %>%
  # subset_samples(Timepoints != "Day 28")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Rotten") %>%
  subset_samples(Cultivar == "Jonathan" | Cultivar == "Gala")  

test_samp <- sample_data(physq_sub)

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Cultivar)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Cultivar, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Cultivar", color="Cultivar", shape="Tissue_types", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Skin Rotten") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Cultivar") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.02, ", paste(italic('p')),"=0.98"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Cultivar", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Skin Day 42", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (69.5%) ", y = "PCoA 2 (15.9%)", colour = "Cultivar", fill = "Cultivar") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```
##Beta Diversity Jonathan vs. Gala Pulp Day 14

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Skin")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints != "Day 42")  %>%
  # subset_samples(Timepoints != "Day 28")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 28") %>%
  subset_samples(Cultivar == "Jonathan" | Cultivar == "Gala")  

test_samp <- sample_data(physq_sub)

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Cultivar)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Cultivar, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Cultivar", color="Cultivar", shape="Tissue_types", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Pulp day 28") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Cultivar") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.79, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Cultivar", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Pulp Day 28", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (82.2%) ", y = "PCoA 2 (10.1%)", colour = "Cultivar", fill = "Cultivar") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

##Beta Diversity Jonathan vs. Gala pulp Day 42

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Skin")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints != "Day 42")  %>%
  # subset_samples(Timepoints != "Day 28")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 42") %>%
  subset_samples(Cultivar == "Jonathan" | Cultivar == "Gala")  

test_samp <- sample_data(physq_sub)

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Cultivar)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Cultivar, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Cultivar", color="Cultivar", shape="Tissue_types", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: pulp day 42") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Cultivar") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.53, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Cultivar", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Pulp Day 42", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (70.4%) ", y = "PCoA 2 (23.4%)", colour = "Cultivar", fill = "Cultivar") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

##Beta Diversity Jonathan vs. Gala skin Day Rotten

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Skin")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints != "Day 42")  %>%
  # subset_samples(Timepoints != "Day 28")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Rotten") %>%
  subset_samples(Cultivar == "Jonathan" | Cultivar == "Gala")  

test_samp <- sample_data(physq_sub)

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Cultivar)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Cultivar, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Cultivar", color="Cultivar", shape="Tissue_types", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Pulp Rotten") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Cultivar") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.16, ", paste(italic('p')),"=0.29"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Cultivar", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Pulp Rotten", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (82.1%) ", y = "PCoA 2 (8.2%)", colour = "Cultivar", fill = "Cultivar") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

##Beta Diversity Jonathan vs. Gala skin Day Rotten

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatment != "Chitosan") %>%
  subset_samples(Treatment != "Chitosanitosan") %>%
  subset_samples(Treatment != "Mock") %>%
  subset_samples(Treatment != "Phage") %>%
  #subset_samples(Cultivar != "Gala") %>%
  subset_samples(Tissue.types != "Pulp")  %>%
  # subset_samples(Timepoints != "Rotten")  %>%
  # subset_samples(Timepoints != "Day 42")  %>%
  # subset_samples(Timepoints != "Day 28")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Rotten") %>%
  subset_samples(Cultivar == "Jonathan" | Cultivar == "Gala")  

test_samp <- sample_data(physq_sub)

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Cultivar)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Cultivar, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Cultivar", color="Cultivar", shape="Tissue_types", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Skin Rotten") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Cultivar") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.02, ", paste(italic('p')),"=0.98"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Cultivar", title="PCoA Weighted Unifrac- Jonathan Vs. Gala: Skin Day 42", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (69.5%) ", y = "PCoA 2 (15.9%)", colour = "Cultivar", fill = "Cultivar") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```




##Beta Diversity by Timepoints in Control PULP vs. skin day 49

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Skin")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 49") %>%
  #subset_samples(Treatments == "ASM-treated")  
  subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue_types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Tissue_types, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue_types", color="Tissue_types", shape="Tissue_types", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue_types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.10, ", paste(italic('p')),"=0.32"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue_types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (62.3%) ", y = "PCoA 2 (19.5%)", colour = "Tissue_types", fill = "Tissue_types") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in ASM-treated vs control Pulp

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 7") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Treatments)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.012, ", paste(italic('p')),"=0.426"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Treatments", fill = "Treatments") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```



#Beta Diversity by Timepoints in ASM-treated vs control Pulp day 0

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA

physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Treatments)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.16, ", paste(italic('p')),"=0.036"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (55.3%) ", y = "PCoA 2 (23.5%)", colour = "Treatments", fill = "Treatments") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in ASM-treated vs control Pulp day 25

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA

physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 25") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Treatments)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.81, ", paste(italic('p')),"=0.002"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (84.2%) ", y = "PCoA 2 (8.6%)", colour = "Treatments", fill = "Treatments") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in ASM-treated vs control Pulp day 49

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 49") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Treatments)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```

#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.03, ", paste(italic('p')),"=0.696"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (82.6%) ", y = "PCoA 2 (11.2%)", colour = "Treatments", fill = "Treatments") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in ASM-treated vs control Pulp day 7

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 7") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Treatments)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.14, ", paste(italic('p')),"=0.12"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (60.4%) ", y = "PCoA 2 (19.2%)", colour = "Treatments", fill = "Treatments") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```
#Beta Diversity by Timepoints in ASM-treated skin and pulp 

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Pulp" | Tissue_types == "Skin")  %>%
  #(Tissue_types == "SKin")  %>%
  #subset_samples(Timepoints == "Day 7") %>%
  subset_samples(Treatments == "ASM-treated") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue_types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Tissue_types, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue_types", color="Tissue_types", shape="Tissue_types", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue_types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.36, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue_types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (61.1%) ", y = "PCoA 2 (20.7%)", colour = "Tissue_types", fill = "Tissue_types") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in control skin and pulp 

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Pulp" | Tissue_types == "Skin")  %>%
  #(Tissue_types == "SKin")  %>%
  #subset_samples(Timepoints == "Day 7") %>%
  subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue_types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Tissue_types, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue_types", color="Tissue_types", shape="Tissue_types", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue_types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.23, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue_types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (67.2%) ", y = "PCoA 2 (14.3%)", colour = "Tissue_types", fill = "Tissue_types") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```




#Beta Diversity by Timepoints in ASM-treated skin and pulp (filter day 49)

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Pulp" | Tissue_types == "Skin")  %>%
  #(Tissue_types == "SKin")  %>%
  #subset_samples(Timepoints == "Day 7") %>%
  subset_samples(Treatments == "ASM-treated") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Tissue_types)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Tissue_types, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Tissue_types", color="Tissue_types", shape="Tissue_types", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue_types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.55, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue_types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (62.8%) ", y = "PCoA 2 (26.1%)", colour = "Tissue_types", fill = "Tissue_types") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```




#Beta Diversity by Timepoints in ASM-treated vs control Skin 

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Skin")  %>%
  # subset_samples(Timepoints == "Day 7") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Treatments)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.10, ", paste(italic('p')),"=0.003"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Treatments", fill = "Treatments") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```


#Beta Diversity by Timepoints in ASM-treated vs control Skin day 49

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Skin")  %>%
  subset_samples(Timepoints == "Day 49") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Treatments)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.04, ", paste(italic('p')),"=0.85"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (65.5%) ", y = "PCoA 2 (15.7%)", colour = "Treatments", fill = "Treatments") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in ASM-treated vs control Skin day 25

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Skin")  %>%
  subset_samples(Timepoints == "Day 25") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Treatments)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.33, ", paste(italic('p')),"=0.004"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (47.9%) ", y = "PCoA 2 (22.8%)", colour = "Treatments", fill = "Treatments") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in ASM-treated vs control Skin day 7

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Skin")  %>%
  subset_samples(Timepoints == "Day 7") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Treatments)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.70, ", paste(italic('p')),"=0.005"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (85.7%) ", y = "PCoA 2 (10.7%)", colour = "Treatments", fill = "Treatments") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in ASM-treated vs control Skin day 0

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  #subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  #subset_samples(Timepoints != "Day 14")  %>%
  #subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  #subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Tissue_types == "Skin")  %>%
  subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control") 

test_samp <- sample_data(physq_sub)
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Treatments)
adonis_wunifrac


## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```



##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.11, ", paste(italic('p')),"=0.17"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (73.3%) ", y = "PCoA 2 (10.5%)", colour = "Treatments", fill = "Treatments") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
pmain_wuF
```

#Beta Diversity by Timepoints in control Skin

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  # #subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.45, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (57.2%) ", y = "PCoA 2 (19.4%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```

#Beta Diversity by Timepoints in control Skin day 0 and day 7

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  subset_samples(Timepoints != "Day 14")  %>%
  subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.29, ", paste(italic('p')),"=0.039"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```

#Beta Diversity by Timepoints in control Skin day 7 and day 14

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  subset_samples(Timepoints != "Day 0")  %>%
  subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.05, ", paste(italic('p')),"=0.50"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```

#Beta Diversity by Timepoints in control Skin day 0 and day 14

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.45, ", paste(italic('p')),"=0.002"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```

#Beta Diversity by Timepoints in control Skin day 0 and day 25

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.10, ", paste(italic('p')),"=0.316"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```


#Beta Diversity by Timepoints in control Skin day 14 and day 25

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Timepoints != "Day 0")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.34, ", paste(italic('p')),"=0.016"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```




#Beta Diversity by Timepoints in control Skin day 0 and day 49

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  subset_samples(Timepoints != "Day 25")  %>%
  subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.30, ", paste(italic('p')),"=0.01"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```

#Beta Diversity by Timepoints in control Pulp

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Skin")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.54, ", paste(italic('p')),"=0.001"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (76.2%) ", y = "PCoA 2 (8.4%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```

#Beta Diversity by Timepoints in control Pulp day 0, day 14 and day 25

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Skin")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  subset_samples(Timepoints != "Day 7")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.19, ", paste(italic('p')),"=0.028"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```

#Beta Diversity by Timepoints in control Pulp day 0 and day 25

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Skin")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  subset_samples(Timepoints != "Day 14")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.07, ", paste(italic('p')),"=0.35"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```


#Beta Diversity by Timepoints in control Pulp day 0 and day 14

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Skin")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  subset_samples(Timepoints != "Day 25")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.23, ", paste(italic('p')),"=0.014"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```


#Beta Diversity by Timepoints in control Pulp day 0 and day 7

##Weighted Unifrac stats 
Analyzed by Timepoints
```{r}
## PERMANOVA
library(vegan)
physq_sub <- physeq_filtered2 %>% 
  # subset_samples(Treatments != "None") %>%
  # subset_samples(Treatments != "ASM-treated")  %>%
  subset_samples(Tissue_types != "Skin")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  subset_samples(Timepoints != "Day 25")  %>%
  subset_samples(Treatments == "Control")
# physeq_filtered3 <- physeq_filtered2 %>%
#   subset_samples(Treatments != "None") 

wuinfrac_dist <- phyloseq::distance(physq_sub, method="wunifrac") #RUN this only once because it takes a lot of time

adonis_wunifrac = adonis2(wuinfrac_dist ~ sample_data(physq_sub)$Timepoints)
adonis_wunifrac
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP

# subset phyloseq object as for sample data
wuni_disp <-betadisper(wuinfrac_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(wuni_disp)
 
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(wuni_disp)

#?plot.betadisper
## Would look better with higher replication for groups
plot(wuni_disp, label = F)

## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(wuni_disp, label = F, hull = F, ellipse = T)

## Within-group dispersion that PERMDISP is testing
boxplot(wuni_disp, las = 2, cex.lab=1.5)
?boxplot

## pairwise p-values
TukeyHSD(wuni_disp)
scores(wuni_disp, 1:4, display = "centroids")
rda(otu_table)
```

##Weighted unifrac beta diversity plot with phyloseq
Used to check the PCoA %
```{r}
beta_wu <- ordinate(physq_sub, "PCoA", "wunifrac")
beta_wu_plot = plot_ordination(physq_sub, beta_wu, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_wu_plot
```


#Final weighted unifrac plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.23, ", paste(italic('p')),"=0.014"))
beta_scatter = as.data.frame(beta_wu[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_wuF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.2%) ", y = "PCoA 2 (19.3%)", colour = "Timepoints", fill = "Timepoints") +annotate("text", x = -0.04, y = -0.06, label = label_perm, colour = "black")
ggtitle("Control: Skin")
pmain_wuF
```



##Bray-Curtis dissimilarity statistics ASM Pulp

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Skin")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Treatments == "ASM-treated")  
  #subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Timepoints)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.46, ", paste(italic('p')),"=0.001"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (42.9%) ", y = "PCoA 2 (18%)", colour = "Timepoints", fill = "Timepoints")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics ASM Skin

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Treatments == "ASM-treated")  
  #subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Timepoints)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.55, ", paste(italic('p')),"=0.001"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (40.9%) ", y = "PCoA 2 (12.5%)", colour = "Timepoints", fill = "Timepoints")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics Control Skin

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  #subset_samples(Treatments == "ASM-treated")  
  subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Timepoints)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Bray") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.29, ", paste(italic('p')),"=0.001"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (28.7%) ", y = "PCoA 2 (19.1%)", colour = "Timepoints", fill = "Timepoints")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```


##Bray-Curtis dissimilarity statistics Control Pulp

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Skin")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  #subset_samples(Treatments == "ASM-treated")  
  subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Timepoints)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA bray") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.31, ", paste(italic('p')),"=0.001"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (37.8%) ", y = "PCoA 2 (14.9%)", colour = "Timepoints", fill = "Timepoints")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics Control Skin Day 0 and day 25

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  subset_samples(Timepoints != "Day 14")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  #subset_samples(Treatments == "ASM-treated")  
  subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Timepoints)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.15, ", paste(italic('p')),"=0.097"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (24.1%) ", y = "PCoA 2 (20.6%)", colour = "Timepoints", fill = "Timepoints")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics Control pulp day 0 to day 25

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Skin")  %>%
  subset_samples(Timepoints != "Day 14")  %>%
  subset_samples(Timepoints != "Day 49")  %>%
  #subset_samples(Timepoints != "Day 25")  %>%
  subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  #subset_samples(Treatments == "ASM-treated")  
  subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Timepoints)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.14, ", paste(italic('p')),"=0.017"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (24.1%) ", y = "PCoA 2 (20.6%)", colour = "Timepoints", fill = "Timepoints")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics Control vs. ASM Skin day 0

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control")  
  #subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Treatments)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Bray") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.06, ", paste(italic('p')),"=0.36"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (53.5%) ", y = "PCoA 2 (13.9%)", colour = "Treatments", fill = "Treatments")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```


##Bray-Curtis dissimilarity statistics Pulp ASM Vs. Control day 0

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Skin")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control")  
  
test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Treatments)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Bray") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.26, ", paste(italic('p')),"=0.013"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (58.1%) ", y = "PCoA 2 (14.5%)", colour = "Treatments", fill = "Treatments")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```




##Bray-Curtis dissimilarity statistics Pulp ASM Vs. Control Day 7

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Skin")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 7") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control")  
  
test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Treatments)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```

##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Bray") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.24, ", paste(italic('p')),"=0.01"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (57.6%) ", y = "PCoA 2 (17.9%)", colour = "Treatments", fill = "Treatments")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics Skin ASM Vs. Control Day 7

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 7") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control")  
  
test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Treatments)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```

##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Bray") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.46, ", paste(italic('p')),"=0.007"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (56.6%) ", y = "PCoA 2 (17.4%)", colour = "Treatments", fill = "Treatments")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```


##Bray-Curtis dissimilarity statistics Skin ASM Vs. Control Day 25

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 25") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control")  
  
test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Treatments)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```

##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Bray") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.24, ", paste(italic('p')),"=0.01"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (32.5%) ", y = "PCoA 2 (20.3%)", colour = "Treatments", fill = "Treatments")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```


##Bray-Curtis dissimilarity statistics Skin ASM Vs. Control Day 49

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 49") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control")  
  
test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Treatments)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```

##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Bray") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.12, ", paste(italic('p')),"=0.72"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (31.4%) ", y = "PCoA 2 (21.1%)", colour = "Treatments", fill = "Treatments")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics Pulp ASM Vs. Control Day 25

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Skin")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 25") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control")  
  
test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Treatments)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```

##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Bray") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.40, ", paste(italic('p')),"=0.001"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (49.9%) ", y = "PCoA 2 (22%)", colour = "Treatments", fill = "Treatments")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics Pulp ASM Vs. Control Day 49

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Skin")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 49") %>%
  subset_samples(Treatments == "ASM-treated" | Treatments == "Control")  
  
test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Treatments)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Treatments, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```

##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Treatments", color="Treatments", shape="Treatments", title="PCoA Bray") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Treatments") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.05, ", paste(italic('p')),"=0.73"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Treatments", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (36.7%) ", y = "PCoA 2 (27%)", colour = "Treatments", fill = "Treatments")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics ASM Skin vs Pulp day 49

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 49") %>%
  subset_samples(Treatments == "ASM-treated")  
  #subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Tissue_types)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Tissue_types, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Tissue_types", color="Tissue_types", shape="Tissue_types", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue_types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.21, ", paste(italic('p')),"=0.041"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue_types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (27.6%) ", y = "PCoA 2 (25.7%)", colour = "Tissue_types", fill = "Tissue_types")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics Control Skin vs Pulp day 49

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  subset_samples(Timepoints == "Day 49") %>%
  #subset_samples(Treatments == "ASM-treated")  
  subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Tissue_types)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Tissue_types, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Tissue_types", color="Tissue_types", shape="Tissue_types", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue_types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.10, ", paste(italic('p')),"=0.33"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue_types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (25.6%) ", y = "PCoA 2 (18.2%)", colour = "Tissue_types", fill = "Tissue_types")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```

##Bray-Curtis dissimilarity statistics Control Skin vs Pulp 

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  #subset_samples(Timepoints == "Day 49") %>%
  #subset_samples(Treatments == "ASM-treated")  
  subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Tissue_types)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Tissue_types, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Tissue_types", color="Tissue_types", shape="Tissue_types", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue_types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.15, ", paste(italic('p')),"=0.001"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue_types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (31.7%) ", y = "PCoA 2 (17.3%)", colour = "Tissue_types", fill = "Tissue_types")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```


##Bray-Curtis dissimilarity statistics ASM Skin vs Pulp 

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  #subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  #subset_samples(Timepoints == "Day 49") %>%
  subset_samples(Treatments == "ASM-treated")  
  #subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Tissue_types)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Tissue_types, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Tissue_types", color="Tissue_types", shape="Tissue_types", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Tissue_types") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.22, ", paste(italic('p')),"=0.001"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Tissue_types", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (33%) ", y = "PCoA 2 (20.8%)", colour = "Tissue_types", fill = "Tissue_types")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```


##Bray-Curtis dissimilarity statistics ASM Skin

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Treatments == "ASM-treated")  
  #subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Timepoints)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.55, ", paste(italic('p')),"=0.001"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (24.1%) ", y = "PCoA 2 (20.6%)", colour = "Timepoints", fill = "Timepoints")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```


##Bray-Curtis dissimilarity statistics ASM Skin

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Treatments == "ASM-treated")  
  #subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Timepoints)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.55, ", paste(italic('p')),"=0.001"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (24.1%) ", y = "PCoA 2 (20.6%)", colour = "Timepoints", fill = "Timepoints")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```


##Bray-Curtis dissimilarity statistics ASM Skin

```{r}
## PERMANOVA
physq_sub <- physeq_filtered2 %>% 
  subset_samples(Treatments != "None") %>%
  subset_samples(Tissue_types != "Pulp")  %>%
  # subset_samples(Timepoints != "Day 14")  %>%
  # subset_samples(Timepoints != "Day 49")  %>%
  # subset_samples(Timepoints != "Day 25")  %>%
  # subset_samples(Timepoints != "Day 7")  %>%
  # subset_samples(Tissue_types == "Pulp")  %>%
  # subset_samples(Timepoints == "Day 0") %>%
  subset_samples(Treatments == "ASM-treated")  
  #subset_samples(Treatments == "Control") 

test_samp <- sample_data(physq_sub)
bray_dist = phyloseq::distance(physq_sub, method="bray") #RUN this only once because it takes a lot of time
adonis_bray = adonis2(bray_dist ~ sample_data(physq_sub)$Timepoints)
adonis_bray
## Significant PERMANOVA indicates that centroid (or spatial median) among groups is different and/or with-group dispersion among groups is different
## PERMDISP
bray_disp <-betadisper(bray_dist, sample_data(physq_sub)$Timepoints, type=c("median"))
anova(bray_disp)
## If PERMANOVA and PERMDISP are both significant, you can use plotting to tell if PERMANOVA was significant based on centroid (or spatial median)
plot(bray_disp)
#?plot.betadisper
## Would look better with higher replication for groups
plot(bray_disp, label = F)
## Plot with 1 standard deviation ellipses around the group medians
## sample size issue here, but you get the idea
plot(bray_disp, label = F, hull = F, ellipse = T)
## Within-group dispersion that PERMDISP is testing
boxplot(bray_disp, las = 2, cex.lab=1.5)
?boxplot
## pairwise p-values
TukeyHSD(bray_disp)
scores(bray_disp, 1:4, display = "centroids")
```


##Bray beta diversity plot
```{r}
beta_bray <- ordinate(physq_sub, "PCoA", "bray") #RUN this only ONCE because it takes a lot of time
beta_bray_plot = plot_ordination(physq_sub, beta_bray, type="Timepoints", color="Timepoints", shape="Timepoints", title="PCoA Weighted Unifrac") + stat_ellipse(type = "t", linetype = 3) + stat_ellipse(type = "t") + theme_bw()+labs(colour = "Timepoints") #To add arrows https://neavemj.github.io/posts/coralMicrobiome
beta_bray_plot
```


##Bray-Curtis final plot
```{r}
label_perm <- expression(paste("PERMANOVA, ",R^2 ,"= 0.55, ", paste(italic('p')),"=0.001"))
#library(lemon)
beta_scatter = as.data.frame(beta_bray[["vectors"]])
beta_meta = merge(beta_scatter,metadata,by = 0,all=F)
pmain_brayF = ggscatter(beta_meta, x = "Axis.1", y = "Axis.2", color = "Timepoints", palette = "aaas",ellipse = TRUE, ellipse.level=.5,mean.point = F, mean.point.size = 5, star.plot = F) +labs(x = "PCoA 1 (24.1%) ", y = "PCoA 2 (20.6%)", colour = "Timepoints", fill = "Timepoints")+ annotate("text", x = -0.12, y = -0.2, label = label_perm, colour = "black")
pmain_brayF
```
##Arranging both diversity plots into one
```{r}
beta_div_plots = ggarrange(pmain_brayF,pmain_wuF,nrow = 2, ncol = 1, align="hv", common.legend = TRUE, labels = c("C","D")) 
beta_div_plots
```

